tasks/jasmine.js

```diff
@@ -194,7 +194,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.display === 'full' || options.showLogging) {
```

Comment:
I'm not sure this is the correct behavior.

|   | display = full (default) | display = short |
|---|---|---|
| showLogging = false (default undefined) | log | silent |
| showLogging = true | log | log |

I would expect something more like:

|   | display = full (default) | display = short |
|---|---|---|
| showLogging = undefined (default) | log | silent |
| showLogging = false | silent | silent |
| showLogging = true | log | log |

Notice how the default value for `showLogging` preserves the existing behavior, but specifying true or false enforces the expected result regardless of `display`.